arch/common/ramfunc.ld

```diff
@@ -9,12 +9,12 @@
 
 SECTION_DATA_PROLOGUE(.ramfunc,,)
 {
-	MPU_ALIGN(_ramfunc_ram_size);
-	_ramfunc_ram_start = .;
+	MPU_ALIGN(__ramfunc_size);
+	__ramfunc_start = .;
 	*(.ramfunc)
 	*(".ramfunc.*")
-	MPU_ALIGN(_ramfunc_ram_size);
-	_ramfunc_ram_end = .;
+	MPU_ALIGN(__ramfunc_size);
```

Comment:
I know this exists before this PR, but can you really read `__ramfunc_size` here before setting it on line 19 below?